pkg/controller/v1beta1/inferenceservice/controller.go

```diff
@@ -278,6 +278,14 @@ func (r *InferenceServiceReconciler) updateStatus(desiredService *v1beta1api.Inf
 		// This is important because the copy we loaded from the informer's
 		// cache may be stale and we don't want to overwrite a prior update
 		// to status with this stale state.
+
+		// Instead, if the model keep running on False state
+		// Raise an event for info of failed conditions.
+		if !wasReady {
+			msg := r.GetFailConditions(desiredService)
```

Comment:
as the msg var is only used here, we can remove it and call the function directly.

Reply:
Okay, thank you for your suggestion. We will make the adjustments.